ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.js

```diff
@@ -271,28 +273,29 @@ function ClustersTablePanel({ selectedClusterId, setSelectedClusterId, searchOpt
                             {messages}
                         </div>
                     )}
-                    {(!fetchingClusters || pollingCount > 0) && currentClusters.length <= 0 && (
-                        <AddClusterPrompt />
-                    )}
-                    {(!fetchingClusters || pollingCount > 0) && currentClusters.length > 0 && (
-                        <div data-testid="clusters-table" className="h-full w-full">
-                            <CheckboxTable
-                                ref={(table) => {
-                                    setTableRef(table);
-                                }}
-                                rows={currentClusters}
-                                columns={clusterColumns}
-                                onRowClick={setSelectedClusterId}
-                                toggleRow={toggleCluster}
-                                toggleSelectAll={toggleAllClusters}
-                                selection={checkedClusterIds}
-                                selectedRowId={selectedClusterId}
-                                noDataText="No clusters to show."
-                                minRows={20}
-                                pageSize={pageSize}
-                            />
-                        </div>
-                    )}
+                    {(!fetchingClusters || pollingCount > 0) &&
+                        currentClusters.length <= 0 &&
```

Comment:
An observation, not a blocker to merge: because a JavaScript array cannot have negative length, I have to pause a few seconds to translate to `currentClusters.length === 0` in the middle of an increasingly complicated condition.